core/templates/dev/head/services/ContextService.js

```diff
@@ -50,9 +54,9 @@ oppia.factory('ContextService', [
       getEditorTabContext: function() {
         var hash = UrlService.getHash();
         if (hash.indexOf('#/gui') === 0) {
-          return EDITOR_TAB_CONTEXT.EXPLORATION_EDITOR;
+          return EXPLORATION_EDITOR_TAB_CONTEXT.EXPLORATION_EDITOR;
```

Comment:
Be careful. This needs to be changed back to EDITOR, I think -- make sure to proofread after submitting.

Also, this suggests that this stuff isn't tested yet. Could you please add Karma tests?

Ditto below/elsewhere -- make sure there is full Karma test coverage please.

Reply:
Sorry for missing these out. CHanged. Added tests with full coverage.